src/VsExtension/OutputConsoleLogger.cs

```diff
@@ -40,6 +42,9 @@ public OutputConsoleLogger(IServiceProvider serviceProvider)
             _buildEvents.OnBuildBegin += (obj, ev) => { ErrorListProvider.Tasks.Clear(); };
             _solutionEvents = dte.Events.SolutionEvents;
             _solutionEvents.AfterClosing += () => { ErrorListProvider.Tasks.Clear(); };
+            var properties = dte.get_Properties("Environment", "ProjectsAndSolution");
+            var value = properties.Item("MSBuildOutputVerbosity").Value;
+            _msBuildOutputVerbosity = value is int ? (int)value : 0;
```

Comment:
What happens if the user changes the logging settings while vs is running? Is this value cached forever here?